SettingsList.py

```diff
@@ -2249,19 +2249,35 @@ def __init__(self, name, gui_text, min, max, default, step=1,
         gui_params     = {
             'randomize_key': 'randomize_settings',
         },
-    ), 
-    Checkbutton(
+    ),
+    Combobox(
         name           = 'mix_entrance_pools',
         gui_text       = 'Mix Entrance Pools',
+        default        = 'off',
+        choices        = {
+            'off':       'Off',
+            'non-ow':    'Non-Overworld',
+            'all':       'All Entrances',
+        },
         gui_tooltip    = '''\
+            'Non-Overworld':
+            Shuffle all interior entrances into a single mixed pool.
+            This means any shuffled entrance could lead to any type
+            of area: Dungeon, Grotto, or Interior. Overworld
+            entrances are kept in a separate pool if randomized.
+    
+            'All Entrances':
             Shuffle all entrances into a single mixed pool.
             This means any shuffled entrance could lead to any type 
             of area: Dungeon, Grotto, Interior or Overworld.
         ''',
-        default        = False,
         shared         = True,
         gui_params     = {
             'randomize_key': 'randomize_settings',
+            'distribution':  [
+                ('off', 1),
+                ('on', 1),
```

Comment:
I assume this is supposed to be 'all'
```suggestion
                ('all', 1),
```

Reply:
Whoops, good catch. Do you want to include the new option here too? I left it out to avoid wider impacts, but it should probably be there.

Reply:
I think we can have it in randomized main rules yeah, while keeping 'off' at 50% chance (eg. off/indoor/all at 2/1/1)

Reply:
Works for me, set to weights of 2/1/1 and tested each could generate.